CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 ## 1.1.3 (Unreleased)
 
 BUG FIXES:
-csi: fixed a CLI panic when formatting `volume status` with `-verbose` flag [[GH-10818](https://github.com/hashicorp/nomad/issues/10818)]
+* csi: Fixed a CLI panic when formatting `volume status` with `-verbose` flag [[GH-10818](https://github.com/hashicorp/nomad/issues/10818)]
+* docker: Moved the generated `/etc/hosts` file's mount source to the allocation directory so that it can be shared between tasks of an allocation. [[GH-10823](https://github.com/hashicorp/nomad/issues/10823)]
```

Comment:
I'd opened this PR before https://github.com/hashicorp/nomad/pull/10825... should I wait to merge and then rebase with the new changelog files?

Reply:
Done